Sources_v3/StreamChatUI/ChatChannel/ChatChannelMessageInputView.swift

```diff
@@ -4,26 +4,40 @@
 
 import UIKit
 
+open class ChatChannelMessageInputTextView: UITextView {
+    override public var text: String? {
+        didSet {
+            delegate?.textViewDidChange?(self)
+        }
+    }
```

Comment:
Just to preserve whatever is in `willSet/didSet` in superclass?
```suggestion
    override public var text: String? {
        get { super.text }
        set { super.text = newValue; delegate?.textViewDidChange?(self) }
    }
```

Reply:
@evsaev `willSet` `didSet` on superclass still work when you override it like @mantergo did:
<img width="523" alt="Screen Shot 2020-12-09 at 10 32 11" src="https://user-images.githubusercontent.com/6388510/101611440-cd650b00-3a09-11eb-9bc5-3262e9a7f6bc.png">